CHANGELOG.md

```diff
@@ -103,6 +103,7 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ### Changed
 
+- `filter_attributes` now imports as `` from `py42.util` instead of `py42.sdk.queries.query_filter`.
```

Comment:
Still implies it is called `filter_attributes` here.  Could do:
`py42.sdk.queries.query_filter` renamed to `py42.util.get_attribute_keys_from_class`.